### DISCUSSION TITLE
Zinius

### DISCUSSION THREAD

**Discussion by Gwynn1** [WoWManiaUK/Redemption#6235] [General]
You are completely and utterly useless

it takes months to find a customer....seconds to lose one

**Comment by GMZinius** [comment]
Firstly: As I said before we do not control or have access to the vote sites #6233
Secondly: There is only a bug when you don't vote correctly 
You would see that I am correct when you do as I advised you in my first reply

1, watch the how to vote on wow-mania youtube channel.
2. To fix the bug don't vote for 24 hours, let the sites reset (VERY IMPORTANT)
3. No need to be aggressive just because you are not hearing what you want to hear.
